tests/quality_pragmaitc/test_goal.py

```diff
@@ -0,0 +1,50 @@
+from goald.quality.pragmatic.model.task import Task
+from goald.quality.pragmatic.model.goal import Goal
+from goald.quality.pragmatic.model.context import Context
+from goald.quality.pragmatic.model.delegation import Delegation
+from goald.quality.pragmatic.model.decomposition import Decomposition
+
+
+def test_shouldBeAchievable():
+    root = Goal(Decomposition.AND)
+
+    context = Context("c1")
+    current = []
+    current.append(context)
+
+    task1 = Task()
+    task2 = Task()
+
+    task1.addApplicableContext(context)
+
+    root.addDependency(task1)
+    root.addDependency(task2)
+
+    plan = root.isAchievable(current, None)
+    assert plan is not None
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/gh/lesunb/p2psimpy/pullRequest?prid=4861978)